### ISSUE TITLE
previous page

### ISSUE THREAD

**Issue by nilsbecker** [nojb/llpp#4]
'delete' to navigate a page up is not a good key on the mac since most macs require awkward fn+backspace combination. also, that is not working for me. may i suggest shift+space?

**Comment by nilsbecker** [comment]
page up and page down work, but they are also not accessible on a macbook keyboard except by pressing the fn+up and fn+down. i think not everybody knows these combinations...

**Comment by nilsbecker** [comment]
still the same. i might add that also the `insert` button is difficult to know on a mobile mac keyboard.

**Comment by nojb** [comment]
Yeah, key bindings in general need a little more work... will work on it, hopefully soon.
